src/Logary/Target.fs

```diff
@@ -108,6 +103,19 @@ module TargetUtils =
       initer = fun metadata -> job {
         let! requests = RingBuffer.create 500u
         let shutdown = Ch ()
+        return { server   = fun _ _ -> loop metadata requests shutdown
+                 requests = requests
+                 shutdown = shutdown
+                 name     = name' }
+      }
+    }
+
+  let willAwareNamedTarget loop name : TargetConf =
```

Comment:
@haf this is where I added a new method for registering "willAware" targets so existing targets can continue to work unaltered. If you can think of a better name, let me know...